src/xcode/ENA/ENA/Source/Services/OTP/__tests__/OTPServiceTests.swift

```diff
@@ -0,0 +1,21 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class OTPServiceTests: XCTestCase {
+
+	func testGIVEN_OTPService_WHEN_retrieveOTP_THEN_OTPIsGenerated() {
+		// GIVEN
+		let otpService = OTPServiceMock()
+
+		// WHEN
+
+
+		// THEN
+
```

Comment:
this test does what? Nothing? If so we can remove it

Reply:
 👍